cmd/kube-apiserver/app/server.go

```diff
@@ -319,6 +322,18 @@ func CreateKubeAPIServerConfig(s *options.ServerRunOptions, nodeTunneler tunnele
 		return nil, nil, nil, nil, nil, err
 	}
 
+	var issuer serviceaccount.TokenGenerator
+	if s.ServiceAccountSigningKeyFile != "" && s.Authentication.ServiceAccounts.IssuerID != "" {
```

Comment:
Do we care about an error message if only one is set?

Reply:
Please review only the last commit.